app/src/main/java/com/example/trashminder/presentation/createdReminder/CustomItem.kt

```diff
@@ -0,0 +1,130 @@
+package com.example.trashminder.presentation.createdReminder
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.trashminder.R
+import com.example.trashminder.model.Reminder
+import com.example.trashminder.presentation.theme.*
+
+@Composable
+fun CustomItem(reminder: Reminder) {
+    if (reminder.type == "plastic") {
```

Comment:
In this case I believe a better practice would be to use 'when' instead of if conditions